server/helpers.js

```diff
@@ -18,3 +18,13 @@ export const validateSignature = ({
 
   return originalSignature === signature;
 };
+
+// 5 minutes tolerance (seconds)
+const TOLERANCE_ZONE = 300;
+
+export const validateTimestamp = (requestTimestamp) => {
+  const currentTimestamp = Math.floor(Date.now() / 1000);
+  const difference = currentTimestamp -  Math.floor(requestTimestamp / 1000);
```

Comment:
do we need to divide them both by 1k? can we instead have the tolerance as `300_000`?

Reply:
Yep I'll do it, normally milliseconds were not that important from what I saw in other implementations. Updated